packages/@aws-cdk/aws-servicecatalog/lib/constraints.ts

```diff
@@ -20,6 +21,36 @@ export interface CommonConstraintOptions {
   readonly description?: string;
 }
 
+/**
+ * Properties for deploying with Stackset, which creates a StackSet constraint.
+ */
+export interface StackSetsConstraintOptions extends CommonConstraintOptions {
+  /**
+   * List of accounts to deploy stacks to.
+   */
+  readonly accounts: string[];
+
+  /**
+   * List of regions to deploy stacks to.
+   */
+  readonly regions: string[];
+
+  /**
+   * IAM role used to administer the StackSets configuration.
+   */
+  readonly adminRole: iam.IRole;
+
+  /**
+   * IAM role used to provision the products in the Stacks.
+   */
+  readonly executionRole: iam.IRole;
+
+  /**
+   * Wether to allow end users to create, update, and delete stacks.
+   */
+  readonly allowStackSetInstanceOperations: boolean;
+}
```

Comment:
Should _all_ of the properties of this interface be required? It seems like we should at least have a default for `allowStackSetInstanceOperations`, no?

Reply:
we don't have an agreed upon default for this, I meant to ask earlier, but is it better to just pick one regardless?

Reply:
dig more digging.  Looks like setting `false` here is a better behavior.  Reason being that changes should be administered on the stackset admin side, otherwise you can have drift in your end user accounts of resources.  Updated that in next commit.